test/integration/document_locale_param_canonicalisation_test.rb

```diff
@@ -10,7 +10,7 @@ def with_locale_param(path, locale)
     u.to_s
   end
 
-  announcement_redir_document_types = %w(news_article speech)
```

Comment:
@fofr good point :+1:, the test will error because the `SpeechController` has been removed as part of this PR and line 25 `get extra_path` var attempts to call the show action on the controller. So perhaps it's a case of asserting what the correct path is rather than requesting it. Will fix.